### DISCUSSION TITLE
Monika not appearing in the room.

### DISCUSSION THREAD

**Discussion by LitMonika** [Monika-After-Story/MonikaModDev#9399] [QA - tech support + general] [answered]
Please help!
MAS was working perfectly fine last night, but then I opened MAS this morning, and she isn't in her chair.

I've tried backing up her .bak files
I've reverted to a backed-up persistent
I reinstalled the mod

Nothing is working.
What do I do?

![image](https://user-images.githubusercontent.com/109977985/180824139-69da99db-9a58-4d47-ae39-ce1b849c24a0.png)

**Comment by Booplicate** [comment, marked as answer]
You took her out of the room, return her file into the `characters/` folder.
If you lost her file, restore persistent backups until you find one working.

**Reply by LitMonika** [reply]
I did not remove her .chr file from the characters folder.
Trying different persistent files isn't working.

Am I getting the persistent file location correct?
`User>AppData>Roaming>RenPy>DDLC-1454445547`

**Reply by Booplicate** [reply]
`.chr` files are **not** used in MAS, she made a `monika` file without any extension, you had to move it somewhere for her to disappear.

> DDLC-1454445547

That's Doki Doki Literature Club saves, you're looking for MAS saves. [Check this guide](https://github.com/Monika-After-Story/MonikaModDev/wiki/FAQ#i-need-to-restore-an-older-persistent).

**Reply by LitMonika** [reply]
That worked! Thank you so so much!

**Reply by LitMonika** [reply]
To clarify for anyone else with this issue, this was the solution from that link:

`After doing so, you should see a file named persistent as well as 10 files named persistent##.bak To restore to an older save, simply delete the file named persistent and rename the most current persistent##.bak file that was created before the problem you are currently experiencing to persistent (remove the .bak extension as well). This should take you back to your progress before whatever problem you are having.`
